commercetools-internal-processors/src/main/java/io/sphere/sdk/annotations/processors/generators/CustomUpdateActionsGenerator.java

```diff
@@ -0,0 +1,44 @@
+package io.sphere.sdk.annotations.processors.generators;
+
+import com.squareup.javapoet.TypeSpec;
+import io.sphere.sdk.annotations.HasCustomUpdateActions;
+
+import javax.annotation.processing.Messager;
+import javax.lang.model.element.Element;
+import javax.lang.model.element.ExecutableElement;
+import javax.lang.model.util.Elements;
+import javax.lang.model.util.Types;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * Generates update action classes for interfaces annotated with {@link io.sphere.sdk.annotations.HasUpdateActions}.
+ */
+public class CustomUpdateActionsGenerator extends AbstractMultipleFileGenerator<ExecutableElement> {
+
+    private final Messager messager;
```

Comment:
👍 Yep since a `messager` is the most appropriate way to show what went wrong while processing the annotation.
Added in this commit.